.gitignore

```diff
@@ -24,4 +24,5 @@ package-lock.json
 .castor*
 coverage
 dist
+packages/*/prototype.js*
```

Comment:
🤔  why `js*`? To support `jsx` just in case?

Reply:
No, to support `.js` and also `.js.map` 😏